generate.js

```diff
@@ -95,7 +95,7 @@ function parseCommandInputs(args, argsByName, errorMessages, targetOutputPathLis
     // Output an error if no targets are defined
```

Comment:
It looks like submitting this file was probably an accident, as it's a generated file, and not a source file.
Can you revert it?

Reply:
My change to this file was to remove the 5 second delay at the end.  I don't think it's necessary is it?  And if not where should I go to remove it?

Reply:
Two things:
1. It's there for my personal debugging purposes, based on nuances of my environment (node and VS don't always cooperate as much as I'd like).
I get that it's not useful to most anybody else though.
2. Changing it in the .js doesn't work. You have to change the .ts or it'll just come back the next time I load the solution.